src/main/java/frc/core/util/PID/PIDTalon.java

```diff
@@ -58,13 +60,22 @@ private void setMasterInverted(boolean isInverted) {
 	}
 
 	public void setFollowers(TalonSRX... followers) {
-		for (TalonSRX follower : followers) {
-			this.followers.add(follower);
-			follower.configFactoryDefault();
-			follower.follow(this.master);
-		}
+		this.followers = this.configFollowers(followers)
+												 .stream()
+												 .collect(Collectors.toList());
+	}
+
+	public List<TalonSRX> configFollowers(TalonSRX... followers) {
+		var configuredFollowers = Arrays.stream(followers)
```

Comment:
tu esqueceu de do **follower.follow(this.master);**

eu achei um pouco confuso com essa nomenclatura do jeito q tu fez... eu tinha pensado em fazer 
```
this.followers = Arrays.stream(followers).collect(Collectors.toList());
this.configFollowers(); // com um foreach
```

mas se tu for fazer assim, com o map, acho q o método faz mais sentido ser algo como "createFollowers", aí tem q add o master do follower e fazer o configFactoryDefault. acho q tu tb n precisa dar o " .stream().collect(Collectors.toList());" como tu fez na linha 63 pq tu já tá retornando uma lista no teu método